llvm/lib/Target/SPIRV/SPIRVCallLowering.cpp

```diff
@@ -461,8 +461,11 @@ bool SPIRVCallLowering::lowerFormalArguments(MachineIRBuilder &MIRBuilder,
                            SPIRV::Extension::SPV_KHR_linkonce_odr)
                    ? SPIRV::LinkageType::LinkOnceODR
                    : SPIRV::LinkageType::Export);
-    buildOpDecorate(FuncVReg, MIRBuilder, SPIRV::Decoration::LinkageAttributes,
-                    {static_cast<uint32_t>(LnkTy)}, F.getGlobalIdentifier());
+    buildOpDecorate(
+        FuncVReg, MIRBuilder, SPIRV::Decoration::LinkageAttributes,
+        {static_cast<uint32_t>(LnkTy)},
+        Function::getGlobalIdentifier(F.getName(), F.getLinkage(),
```

Comment:
Can this just use F.getName() ? Not clear why they would need the global identifier with the file name. Also, I found this was added in https://reviews.llvm.org/D116464, which also added a call to getGlobalIdentifier from SPIRVInstructionSelector::selectGlobalValue, and that has since been changed to use getName().

Reply:
Changed to `F.getName()`